modules/bitgo/src/v2/coins/near.ts

```diff
@@ -269,13 +322,283 @@ export class Near extends BaseCoin {
       txHex: serializedTx,
     } as any;
   }
+  // TODO(BG-51092): Needs to be moved to a common place to re-use it for other coins
+  static async initializeMPCIfNotPresent() {
+    if (!this.initialized) {
+      const hdTree = await Ed25519BIP32.initialize();
+      this.MPC = await Eddsa.initialize(hdTree);
+      this.initialized = true;
+    }
+  }
```

Comment:
nit: this construction is unsafe by default. Instead of trusting all places where initialization is needed to call this function first, why not have this return the `MPC` instance and perform this initialization if needed?
```suggestion
  static async getInitializedMpcInstance(): MPC {
    if (this.initialized) {
      return this.MPC;
    }
    const hdTree = await Ed25519BIP32.initialize();
    this.MPC = await Eddsa.initialize(hdTree);
    this.initialized = true;
    return this.MPC;
  }
```

Reply:
Definitely a better approach. Thanks!

Reply:
Done